x-pack/test/alerting_api_integration/observability/metric_threshold_rule.ts

```diff
@@ -86,7 +86,7 @@ export default function ({ getService }: FtrProviderContext) {
             criteria: [
               {
                 aggType: Aggregators.AVERAGE,
-                comparator: Comparator.GT,
+                comparator: COMPARATORS.GREATER_THAN,
```

Comment:
@fkanout What do you think about using the legacy comparator here, just to make sure it works as before?

Reply:
We no longer provide the legacy outside range as an option to create the rule.